big_tests/tests/service_mongoose_system_stats_SUITE.erl

```diff
@@ -101,6 +110,13 @@ user_stats_are_reported_to_google_analytics_when_mim_starts(_Config) ->
     all_event_have_the_same_client_id(),
     ok.
 
+system_stats_are_not_reported_when_not_allowed(_Config) ->
+    trigger_reporting(mim3()),
+    timer:sleep(1000),
```

Comment:
This sleep makes me nervous, sleeps always eventually betray you on CI. Can we rearrange the logic in a wait_until kind of way? 😅

Reply:
modified to check if the process is still registered